kura/org.eclipse.kura.rest.service.listing.provider/src/main/java/org/eclipse/kura/internal/rest/service/listing/provider/dto/InterfacesIdsDTO.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.kura.internal.rest.service.listing.provider.dto;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.kura.KuraErrorCode;
+import org.eclipse.kura.KuraException;
+
+public class InterfacesIdsDTO {
+
+    private final List<String> interfacesIds;
```

Comment:
You can try changing this to Set<String>, in this way duplicates are removed from request automatically.